silverback/_cli.py

```diff
@@ -152,12 +148,22 @@ def build(generate, path):
             bots.append(file)
         for bot in bots:
             dockerfile_content = DOCKERFILE_CONTENT
+            if (Path.cwd() / "requirements.txt").exists():
+                dockerfile_content += "COPY requirements.txt .\n"
+                dockerfile_content += (
+                    "RUN pip install --upgrade pip && pip install -r requirements.txt\n"
+                )
+
+            if (Path.cwd() / "ape-config.yaml").exists():
+                dockerfile_content += "COPY ape-config.yaml .\n"
+                dockerfile_content += "RUN ape plugins install -U .\n"
+
             if "__init__" in bot.name:
```

Comment:
do I have to put `__init__` in my path, or will it autodetect a `bot/` folder upstream of this?

Reply:
We do not autodetect a `bot/` folder upstream currently. If you want to use that generate command, it assumes the `bots/` folder exists based on the default path we give it. 

I need to rework this generate command it looks like. I don't think it does what we want from what I'm looking at.

We want this to do this:

- look for a `bots/` folder. Build a dockerfile for each bot file in there
- If the `bots/` folder does not exist, we want to check for a `bot/` folder. 
- If a `bot/` folder does not exist, check for a `bot.py` file

Unless a user gives a different value for the path. Correct?

Reply:
yes, that sounds correct. I think in the 2nd and 3rd cases it should generate `.silverback-images/Dockerfile.bot`

Reply:
I did `.silverback-images/Dockerfile.<repo>-bot` based on the conversation we had outside of this chat space. Is that correct?

Reply:
that sounds fine. I don't think we will end up with a problem one way or another, since there's only one bot

for the tag, I was hoping it would be either `ghcr.io/<org>/<repo>-bot` or `ghcr.io/<org>/<repo>-<bot>` if the bot has a name (e.g. is under the `bots` folder)

Reply:
Ohhh, I like that. Will be implemented tomorrow